apps/src/sites/studio/pages/programming_classes/show.js

```diff
@@ -30,7 +37,11 @@ $(document).ready(() => {
           categoriesForNavigation={categoriesForNavigation}
           currentCategoryKey={currentCategoryKey}
         >
-          <ProgrammingClassOverview programmingClass={programmingClass} />
+          <ProgrammingClassOverview
```

Comment:
(minor, for future) Just a suggestion, this seems like a place where React.Context might be useful to pass the programming environment data down the component tree.